package.json

```diff
@@ -14,7 +14,8 @@
         "test": "jest",
         "test:coverage": "npx jest --ci --coverage",
         "build": "tsc",
-        "lint": "eslint --ext .ts"
+        "lint": "eslint --ext .ts",
+        "watch": "tsc --watch"
```

Comment:
This is a very interesting addition for the development process of this SDK. 

From the documentation, it looks like this is mostly used to restart applications or servers whenever a new change is made in the code. I'm wondering how the rest of the team can use this for this SDK, since you'd need an application/program that uses the SDK to test the latest changes. Maybe it's something that can be improved in my testing workflow, which I'm curious to find out.

Reply:
You run this command locally. It hangs a watcher on whole project files and re-builds every time some file is changed.

I usually work with such a sdk development (when you have some local package and there is a local client that uses this package) using linking.

1. Create a link in your sdk `npm link`
2. Run watcher `npm run watch`
3. Go to the client folder and link the local sdk package `npm link @1password/connect`
4. Make some changes in sdk. Cause of watcher it rebuilds every time therefore in the client you have the latest sdk version.
5. Run client to test sdk changes

Hope that makes sense :)

Reply:
Oooooh so basically it helps to automatically bring the latest local version of the sdk into the js application that uses it. 

Thank you so much for this info. This will definitely save me time while working on it (from running multiple commands to only 1-2, which is awesome).